resources/public/api/conf/1.0/application.raml

```diff
@@ -10,17 +10,17 @@ documentation:
  - title: Overview
    content: !include docs/overview.md
  - title: Versioning
-   content: !include https://developer.service.hmrc.gov.uk/api-documentation/assets/common/docs/versioning.md
+   content: !include https://developer.service.hmrc.gov.uk/api-documentation/assets/uk.gov.hmrc.testuser.common/docs/versioning.md
  - title: Errors
-   content: !include https://developer.service.hmrc.gov.uk/api-documentation/assets/common/docs/errors.md
+   content: !include https://developer.service.hmrc.gov.uk/api-documentation/assets/uk.gov.hmrc.testuser.common/docs/errors.md
 
 mediaType: [ application/json, application/hal+json ]
 
 uses:
-  sec: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/securitySchemes.raml
-  headers: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/headers.raml
-  annotations: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/annotations.raml
-  types: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/types.raml
+  sec: https://developer.service.hmrc.gov.uk/api-documentation/assets/uk.gov.hmrc.testuser.common/modules/securitySchemes.raml
+  headers: https://developer.service.hmrc.gov.uk/api-documentation/assets/uk.gov.hmrc.testuser.common/modules/headers.raml
+  annotations: https://developer.service.hmrc.gov.uk/api-documentation/assets/uk.gov.hmrc.testuser.common/modules/annotations.raml
+  types: https://developer.service.hmrc.gov.uk/api-documentation/assets/uk.gov.hmrc.testuser.common/modules/types.raml
```

Comment:
I don't think these should have changed.

Reply:
I think I agree